main/zoe/api/zoe.md

```diff
@@ -175,16 +216,20 @@ should carefully consider the impact before sharing access to the `creatorFacet`
 `creatorInvitation` is an invitation that the contract instance creator can use. 
 It is usually used in contracts where the creator immediately sells 
 something (auctions, swaps, etc.), so it's helpful for the creator to have 
-an invitation to escrow and sell goods.
+an invitation to escrow and sell goods. Remember that Zoe invitations are 
+represented as a `payment`.
+
+**tyg todo: In the example call below, should the first argument to startInstance
+be better named "installation" or "installationHandle" i.e. should you call it with the object or a reference, or does it make a difference?**
```

Comment:
the first parameter of startInstance() is an installation object as described above.